buildSrc/src/main/kotlin/Project.kt

```diff
@@ -39,8 +39,7 @@ object Dependencies {
   const val mapboxGlNative = "com.mapbox.maps:android-core:${Versions.mapboxGlNative}"
   const val mapboxCoreCommon = "com.mapbox.common:common:${Versions.mapboxCommon}"
   const val mapboxOkHttp = "com.mapbox.common:okhttp:${Versions.mapboxCommon}"
-  const val mapboxAndroidCore = "com.mapbox.mapboxsdk:mapbox-android-core:${Versions.mapboxAndroidCore}"
-  const val mapboxAndroidTelemetry = "com.mapbox.mapboxsdk:mapbox-android-telemetry:${Versions.mapboxAndroidTelemetry}"
+//  const val mapboxAndroidCore = "com.mapbox.mapboxsdk:mapbox-android-core:${Versions.mapboxAndroidCore}"
```

Comment:
Remove?

Reply:
We need to revert this one back as we had api dependency on the mapboxAndroidCore, otherwise it would be a breaking change. Will actually revert it now..